src/sb-util.ts

```diff
@@ -19,6 +19,7 @@ import {
 
 import { map, filter, makeIterable, first } from './generators';
 import { BlockOpcodeToShape } from './block-shapes';
+export { BlockShapes } from './block-shapes';
```

Comment:
can you leave a note here that this export might change location/name and shouldn't be considered "public" API quite yet ?

I would expect the `BlockOpcodeMeta` concept to hold more than just shapes and what not some day, and want a better api to get into it. 

Works as is for now though, just don't want to find ourselves relying on specificly this key in sb-util's general index export